corporate/lib/stripe.py

```diff
@@ -2305,6 +2305,16 @@ def get_next_plan(self, plan: CustomerPlan) -> Optional[CustomerPlan]:
             ).first()
         return None
 
+    def get_annual_recurring_revenue_for_support_data(
+        self, plan: CustomerPlan, last_ledger_entry: LicenseLedger
+    ) -> int:
+        if plan.fixed_price is not None:
+            # For support and activity views, we want to show the annual
+            # revenue for the currently configured fixed price, which
+            # is the annual amount charged in cents.
+            return plan.fixed_price
```

Comment:
Doesn't this depend on the billing cycle? I think fixed-price with monthly billing may be a possible variation.

Also, what's the thinking around whether we should instead be changing `get_customer_plan_renewal_amount`?

Reply:
Fixed price monthly billing is possible, but the amount in the `fixed_plan` field of the model is the annual amount in cents. That would probably be useful documentation to add. But my previous changes were incorrect, in that the non-fixed price plans were returning an amount based on the billing schedule (monthly or annually).

I've updated the new helper function to always return the annual revenue amount.

---

Re: why add a new function instead of updating `get_customer_plan_renewal_amount` is that for user-facing billing pages, we want to show this:

```python
        if plan.fixed_price is not None:
            if plan.end_date == self.get_next_billing_cycle(plan):
                return 0
```

And for activity/support pages, we want to show the annual fixed price amount.